wof/WaterML_1_1.py

```diff
@@ -389,7 +389,7 @@ def quote_xml(inStr):
     "Escape markup chars, but do not modify CDATA sections."
     if not inStr:
         return ''
-    s1 = (isinstance(inStr, basestring) and inStr or
+    s1 = (isinstance(inStr, str) and inStr or
```

Comment:
Note that I don't have any stake in this so when I say that in almost 2019 and going all in for Python 3 is :+1: take it with a grain of salt.

If Python 2 and 3 support is needed you may look into tools like `six` and lines like this one, for example, would be `six.string_types` instead.